module5/labs/Week1/ErrorHandling.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: page
+title: Error Handling Lab
+---
+
+In today's lab, we are going to continue adding Error Handling to our MVC Movies Application.
+
+### Step 1: Adding Null Checking
+
+First check out a new branch. Then add null checking throughout the rest of the application.
+
+Once you think your application is thoroughly null-checked, create a PR and have a code review session with a peer.
+
+### Step 2: Adding Validation
+
+First check out a new branch. Then add validations throughout the rest of the application.
+
+Once you think your application is thoroughly validated, create a PR and have a code review session with a peer.
+
+### Step 3: Sad Path Testing
+
+First check out a new branch. Then spend five minutes researching "Sad path testing". Send a description of sad path testing in your own words to all instructors.
+
+Then add two "Sad path" tests to your application.
+
+Once you have added your tests, create a PR and have a code review session with a peer.
+
+
+<aside class="instructor-notes" markdown="1">
+    <p><strong>Instructor Note</strong><br>
+    If students finish early and are looking for more to do, you could point them towards researching/implementing a global exception handler or a custom error page.</p>

```

Comment:
I think it will be interesting to see how this timing plays out.  I think there is enough space in the lab that it won't be a catastrophe if the lesson goes over into the afternoon.